tests/examples.ipynb

```diff
@@ -0,0 +1,642 @@
+{
```

Comment:
Rename the folder to `examples` if you intend to put examples in it.
A `tests` or `test` folder will be used for tests

Reply:
Thanks, I'll move it to the examples folder.